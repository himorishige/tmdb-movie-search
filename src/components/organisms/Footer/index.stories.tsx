import { Story, Meta } from '@storybook/react';
import Footer from '.';

export default {
  title: 'TMDB Search/Organisms/Footer',
  component: Footer,
} as Meta;

const Template: Story = (args) => <Footer {...args} />;

export const Default = Template.bind({});
