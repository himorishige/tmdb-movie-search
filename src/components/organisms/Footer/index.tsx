import { memo } from 'react';
import { Flex, Text } from '@chakra-ui/react';

const Footer: React.VFC = memo(() => {
  return (
    <Flex alignItems="center" justifyContent="center" p="1rem" bgColor="gray.800">
      <Text color="white">copyright 2021 &copy;someOneElse</Text>
    </Flex>
  );
});

export default Footer;
