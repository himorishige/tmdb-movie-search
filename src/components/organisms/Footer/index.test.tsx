import renderer from 'react-test-renderer';
import Footer from '.';

describe('Organisms/Footer', () => {
  test('正しくレンダリングされている', () => {
    const footer = renderer.create(<Footer />).toJSON();
    expect(footer).toMatchSnapshot();
  });
});
