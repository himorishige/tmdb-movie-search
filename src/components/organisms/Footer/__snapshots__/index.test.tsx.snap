// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Organisms/Footer 正しくレンダリングされている 1`] = `
<div
  className="css-1mttay7"
>
  <p
    className="chakra-text css-kbn7if"
  >
    copyright 2021 ©someOneElse
  </p>
</div>
`;
